assets/src/edit-story/masks/karma/display.karma.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import WithMask from '../display';
+
+describe('first test', () => {
+  it('should not work', () => {
+    const element = {
```

Comment:
To avoid duplicating test data, it would be useful to have a library of entity factories such as 'createElement', eg:
https://github.com/google/web-stories-wp/pull/1734/files#diff-ee079b61929c70174f7bfee704f0e407R17

Reply:
Agree. Or, alternatively, somewhere I saw folk would create a `__db__` folder and dump actual JSON objects as separate modules that can be then used with factories like `createElement`.

However, this is not how I think we'll go in long-term on elements themselves. Instead, see the [test in this pull request](https://github.com/google/web-stories-wp/pull/1697/files#diff-31cc70608c343a4c8e96b44445ab95a8). Instead here, I actually execute our main `useInsertElement` hook directly to insert any element type I need. The benefit here is that the cost of calling it is low, but we get to invoke all the right element decoration routines to get the right element state right away.